########vorlage



#Postition To write Request
string MyInput










---

string MyOutput


#Position To write Response
























##
